mongodb/vibe/db/mongo/connection.d

```diff
@@ -746,6 +774,8 @@ struct ServerDescription
 	Nullable!int setVersion;
 	Nullable!BsonObjectID electionId;
 	string primary;
+	bool secondary;
+	bool ismaster;
```

Comment:
I couldn't find this in the [documentation](https://www.mongodb.com/docs/v6.0/reference/command/hello/), is this an undocumented field, or am I just looking in the wrong place?